src/pybind/mgr/cephadm/nfs.py

```diff
@@ -0,0 +1,96 @@
+import json
+import logging
+
+from typing import Dict, Optional
+
+import cephadm
+from orchestrator import OrchestratorError
+
+logger = logging.getLogger(__name__)
+
+class NFSGanesha(object):
+    def __init__(self,
+                 mgr,
+                 daemon_id,
+                 pool,
+                 namespace=None):
+        # type: (cephadm.CephadmOrchestrator, str, str, Optional[str]) -> None
+        self.daemon_type = 'nfs'
+        self.daemon_id = daemon_id
+
+        self.mgr = mgr
+
+        # rados pool config
+        self.pool = pool
+        self.namespace = namespace
+
+    def get_daemon_name(self):
```

Comment:
This can be removed.

Reply:
this is also used for building the rados config obj name (`get_rados_config_obj`)

think I'd rather retain this for the off chance we change the format for a daemon name.

Reply:
> this is also used for building the rados config obj name (`get_rados_config_obj`)
> think I'd rather retain this for the off chance we change the format for a daemon name.


It is fine if the format is changed. Otherwise we can remove it and directly use  `get_rados_user()`. Even if `get_rados_config_obj` uses it.